src/elli/dispersions/table_epsilon.py

```diff
@@ -25,7 +27,7 @@ class TableEpsilon(Dispersion):
         The interpolation in the given wavelength range.
     """
 
-    single_params_template = {"lbda": None, "epsilon": None}
+    single_params_template = {"lbda": None, "epsilon": None, "kind": "linear"}
```

Comment:
I wouldn't insert this via the single_params as this is not really a fitting parameter and we may use this dict to update the fitting parameters at some point (which is not really relevant for the table but I think we should still keep it consistent).

I would rather add it to the `__init__` directly, i.e., adding something like
```
self.kind = kwargs.get('kind', 'linear')
```
in the `__init__` or directly as a parameter.

Reply:
ok good. i was not completely sure if all other kwargs get propagated or are ignored.